libraries/adaptive-expressions/src/builtinFunctions/indexOf.ts

```diff
@@ -0,0 +1,50 @@
+/**
+ * @module adaptive-expressions
+ */
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { ExpressionEvaluator } from '../expressionEvaluator';
+import { Expression } from '../expression';
+import { ReturnType } from '../returnType';
+import { ExpressionType } from '../expressionType';
+import { FunctionUtils } from '../functionUtils';
+import { MemoryInterface } from '../memory/memoryInterface';
+import { Options } from '../options';
+
+/**
+ * Return the starting position or index value of a substring. This function is case-insensitive, and indexes start with the number 0.
```

Comment:
>or index value of a substring [](start = 32, length = 29)

of a value in a string.
What is returned for not present?  
What if the value is found more than once?